packages/web3-circle-sdk/src/FaucetApi.ts

```diff
@@ -6,11 +6,11 @@ export class FaucetApi extends BaseApi {
    * Request testnet tokens for your wallet.
    * https://developers.circle.com/api-reference/w3s/programmable-wallets/request-testnet-tokens
    * @param params the parameters for the faucet request
-   * @returns boolean
+   * @returns void
+   *
+   * TODO: this endpoint doesn't work, returns TypeError: Only absolute URLs are supported
```

Comment:
fyi @avkos

Reply:
but it works in tests.
```
describe('Faucet Api', () => {
  const faucetApi = new FaucetApi(apikey, BASE_URL);

  it('Request native token', async () => {
    const res = await faucetApi.request({
      blockchain: BLOCKCHAIN.ETH_SEPOLIA,
      address: ETH_SEPOLIA_WALLET_ADDRESS,
      native: true,
    });
    expect(res).toBe(true);
  });

  it('Request eurc token', async () => {
    const res = await faucetApi.request({
      blockchain: BLOCKCHAIN.ETH_SEPOLIA,
      address: ETH_SEPOLIA_WALLET_ADDRESS,
      eurc: true,
    });
    expect(res).toBe(true);
  });

  it('Request usdc token', async () => {
    const res = await faucetApi.request({
      blockchain: BLOCKCHAIN.ETH_SEPOLIA,
      address: ETH_SEPOLIA_WALLET_ADDRESS,
      usdc: true,
    });
    expect(res).toBe(true);
  });
});
```

Reply:
our sdk return boolean

Reply:
I tested it from just a simple node script and faucet works fine, so it must be something with remix then, I will investigate, sorry. 
you can keep boolean return, there was a bug with 'wallet' part though in here, no?

```
------------------------------------------------------ ⬇️ (unnecessary)
await this.postRequest<void>('/faucet/drips', params, 'wallets')
```